admin/tabs/general/roles/RoleModel.ts

```diff
@@ -207,6 +221,32 @@ export class RoleModel extends HoistModel {
         });
     }
 
+    private processRolesForTreeGrid(
+        roles: Omit<HoistRole, 'users' | 'directoryGroups' | 'roles'>[]
+    ) {
+        const topList = [];
```

Comment:
I think this method would benefit from some more specific variable names, particularly `list` and `topList`, which feel pretty ambiguous to me.  Maybe `ret` or `root` instead of `topList` and `children` instead of `list`?